ecs_prometheus.tf

```diff
@@ -40,7 +40,7 @@ resource "aws_ecs_task_definition" "prometheus" {
     "logConfiguration": {
       "logDriver": "awslogs",
       "options": {
-        "awslogs-group": "${data.terraform_remote_state.management.outputs.ecs_cluster_main_log_group.name}",
+        "awslogs-group": "${aws_cloudwatch_log_group.monitoring.name}",
```

Comment:
Does this give us a monitoring log group, with /monitoring/$service sub groups? I’d prefer to be able to go directly to the logs of each service that runs, than a glob of all the services logs.